recipes-app/iot2050-firmware-update/files/iot2050-firmware-update.tmpl

```diff
@@ -663,19 +684,24 @@ def main(argv):
                         help='Specify the firmware backup directory',
                         nargs=1,
                         default=os.getenv('HOME'))
+    parser.add_argument('-q', '--quiet',
+                        help='Update firmware quietly, assume "yes" as answer to \
+                        all promptas except reboot is "no" and run non-interactively',
```

Comment:
Tell  the user that this would not auto reboot the device after upgrading.

Reply:
except reboot is 'n' is telling this message. You were suggesting to write it more detailed?

Reply:
s/promptas/prompts/

Reply:
> `except reboot is 'n'` is telling this message. You were suggesting to write it more detailed?

I think it's better to describe the behavior from user perspective, instead of from the tool perspective, so I would write something like `Update firmware quietly with the old firmware auto backed up and rollback in case of failure, and without automatic rebooting.`